apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingDelegateChanges/StakingDelegateChanges.constants.tsx

```diff
@@ -51,6 +52,7 @@ export const COLUMNS_CONFIG = [
       <TransactionIdRenderer
         hash={item.id.split('-')[0]}
         dataAttribute="staking-delegate-history-tx-hash"
+        chainId={getCurrentChain()}
```

Comment:
shouldn't it be the value from `useCurrentChain()`?

Reply:
It returns same thing, just that `getCurrentChain()` is not reactive. Can be used inline so don't need to have extra variable.